.env.template

```diff
@@ -14,4 +14,5 @@ NEXTAUTH_SECRET=
 ANALYZE=true|false
 
 # === Dummy feature flags ===
-NEXT_PUBLIC_USE_PORTABLE_TEXT=true|false
\ No newline at end of file
+NEXT_PUBLIC_USE_PORTABLE_TEXT=true|false
+NEXT_PUBLIC_USE_COLLIBRA_TEST=true|false
```

Comment:
i would suggest a more descriptive name for this. think about what we're actually trying to complete with having this flag.

Reply:
Hi sure will change name ...Also i  thought wrt user test it need to get restricted . will add this condition at access.tsx

Reply:
yes, user test is the short term use case, but we generally want variable names want to describe what they're accomplishing in the program. so something like `PREVENT_WORKFLOW_TRIGGER` or `FULFILL_CHECKOUT`

Reply:
Updated PR